recipes/gettext/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment, tools
+import os
+import shutil
+import glob
+
+
+class GetTextConan(ConanFile):
+    name = "gettext"
+    description = "An internationalization and localization system for multilingual programs"
+    topics = ("conan", "gettext", "intl", "libintl", "i18n")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gettext"
+    license = "GPL-3.0-or-later"
+    settings = "os_build", "arch_build", "compiler"
+    exports_sources = ["patches/*.patch"]
+
+    requires = [("libiconv/1.15", "private")]
```

Comment:
```suggestion
    build_requires = [("libiconv/1.15", "private")]
```

Reply:
IIUC conan has no notion of a private build requirements. It seems to me that private requires is more correct than a public build_requires

Reply:
Yes, keep requirement as private, much better. Thanks!